design-documents/testing/static/coding-standard-versioning.md

```diff
@@ -25,12 +25,21 @@ Since we want all users (core developers, community contributors, system integra
         ...
     }
 ```
+## Release cadence
+New versions will be released monthly (if there are changes since the last release), as well as with each Magento patch/minor release. They will include all the latest changes to the checks. There will be three distinct cases that will differentiate what changes they introduce:
```

Comment:
That sounds good to me, however we do not cover here critical bug fixes and how we will deliver it.
I suggest adding something like:

- Critical bugfixes may be delivered earlier to unblock affected teams.

Reply:
Would this be a better description?

Cadence plan:
	1.	Critical fixes (broken suite, broken sev-10 tests) deployed as ready in isolation 
	2.	Monthly releases that include:
	a.	Bugfixes
	b.	New non-failing tests (warning only)
	c.	(future) no changes in code quality scores – if new tests have been added, they need to be excluded from scoring
	3.	Magento-bundled releases that include:
	a.	Patch releases: all the above + non-failing (sev 9 and below) severity promotions/demotions, and changes to scoring
	b.	Minor releases: all the above + new failing tests can be introduced